lib/rules/no-jquery-methods.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @fileOverview Disallow the use of specific jQuery methods.
+ * @author Marc Lynch
+ */
+'use strict';
+
+/**
+ * Determines if this expression matches the method blacklist invocation.
+ * @param {ASTNode} node The identifier node.
+ * @returns {Boolean} Returns true if the expression matches, otherwise false.
+ */
+
+const BLACKLIST = [
```

Comment:
I'm going to remove the hardcoded `const BLACKLIST...` from the rule and instead have the BLACKLIST passed as a param `.eslintrc.js` (context.options)